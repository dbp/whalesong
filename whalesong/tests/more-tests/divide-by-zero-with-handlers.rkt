#lang whalesong

(with-handlers ([exn:fail?
                 (lambda (exn)
                   (printf "I see ~s\n" (exn-message exn)))])
  (/ 1 0))
