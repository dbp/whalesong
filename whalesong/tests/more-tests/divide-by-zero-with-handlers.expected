I see "/: division by zero"
